application/src/main/java/org/togetherjava/tjbot/commands/help/AskCommand.java

```diff
@@ -55,10 +59,10 @@ public final class AskCommand extends SlashCommandAdapter {
      * @param config the config to use
      * @param helper the helper to use
      */
-    public AskCommand(@NotNull Config config, @NotNull HelpSystemHelper helper) {
+    public AskCommand(@NotNull Config config, @NotNull HelpSystemHelper helper, Database database) {
```

Comment:
You forgot to also adjust `ImplicitAskListener`, which also creates help threads.

At that point, instead of duplicating the logic twice, it should be moved into `HelpSystemHelper`.